scripts/update-bq.ts

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable no-console */
+
+import { BigQuery } from '@google-cloud/bigquery'
+import jsonData from '../src/data/mainnet/tokens-info.json'
+
+const bigquery = new BigQuery()
+
+const projectId = 'celo-mobile-mainnet' // we don't index testnet data
+const datasetId = 'address_metadata'
+const tableId = 'tokens_info'
+const fieldsToKeep = ['address', 'decimals', 'name', 'symbol'] as const
```

Comment:
is the `as const` required?

Reply:
no

Reply:
wait actually yes it is. lol. line 16 gets a type error without it

Reply:
TS is very smart about the json import and "entry" is inferred to have this type:
```typescript
entry: {address: string, decimals: number, imageUrl: string, name: string, symbol: string, isSwappable?: undefined, minimumAppVersionToSwap?: undefined, isCoreToken?: undefined, pegTo?: undefined, isSupercharged?: undefined} | ... 8 more ... | {}
```

Reply:
so in the forEach callback, having "field" with a `fieldsToKeep` union type is helpful, because TS knows those are valid keys of `entry`